.gitignore

```
@@ -114,5 +114,8 @@ ENV/
 # Chaste_cg specific
 chaste_codegen/data/tests/chaste_reference_models/develop/
 
+#Mac
+**DS_Store
```

```suggestion
.DS_Store
```

`**DS_Store` is probably fine. It could potentially match files we didn't intend, though.

---

Are you okay for me to change it @mirams?

---

Maybe **.DS_Store ? (I think you need the ** for 'within any subdirectory' ?)

---

I think using just `.DS_Store` also covers `**/.DS_Store` ([https://stackoverflow.com/questions/107701](https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository))